### ISSUE TITLE
Android Support

### ISSUE THREAD

**Issue by Juesto** [Noitidart/Chrome-Store-Foxified#7]
Preview of chrome store in chrome:
![screenshot_2015-10-21-20-11-31](https://cloud.githubusercontent.com/assets/12691160/10652858/4ea3fe8e-7830-11e5-962f-51b7b710d9d4.png)

Webstore in Nightly:
![screenshot_2015-10-21-20-17-05](https://cloud.githubusercontent.com/assets/12691160/10652932/dd65dac0-7830-11e5-99ab-61d9cffe57ea.png)

**Comment by Noitidart** [comment]
High priority for v1.2

**Comment by Juesto** [comment]
Later because we have to wait until fennec webext lands

**Comment by Noitidart** [comment]
no longer applicable in webextensions, the webextension should work in android automatically.

I'll leave this open though until someone confirms it.

**Comment by kahing** [comment]
Is there store intended to work on Android? I get essentially the same thing even when I visit with Android chrome

**Comment by Juesto** [comment]
we're extending what chrome doesnt do on firefox webexts
chrome does not support their store stuff

**Comment by Noitidart** [comment]
@kahing yep Juesto is right we intend to support Android. I didn't test on Android. Someone posted a review saying they had to change a user agent to get it to work. I'll try to implement the steps they took into v3.4

**Comment by Juesto** [comment]
Correct, a user agent hack is what makes this addon tick basically

**Comment by Noitidart** [comment]
What shold I change the user agent too?

**Comment by Juesto** [comment]
likely, don't you do that already?

**Comment by Noitidart** [comment]
No i don't. Its not needed on desktop.

**Comment by Noitidart** [comment]
@kahing @Juesto please try this one, it should change the user agent so you dont have to. 
[webext.zip](https://github.com/Noitidart/Chrome-Store-Foxified/files/1497096/webext.zip)

**Comment by Noitidart** [comment]
@kahing I just uplaoded v3.4 which has the "untested fix" for Android. May you please try out this version and let us know if you can access the store.
